src/actions/guides/getting-started/installing.cr

```diff
@@ -29,51 +27,127 @@ class Guides::GettingStarted::Installing < GuideAction
     [forego](https://github.com/ddollar/forego#installation),
     or [foreman](https://github.com/ddollar/foreman#installation).
 
-    > By  default Lucky creates a `Procfile.dev` that  defines  what processes should be started when running `lucky dev`.
+    > By default Lucky creates a `Procfile.dev` that  defines  what processes should be started when running `lucky dev`.
     > You can modify the `Procfile.dev` to start other processes like running
     > background jobs.
 
-    ### Postgres database
+    ## Postgres database
+
+    ### Install Postgres
 
     Lucky uses Postgres for its database. Install Postgres ([macOS](https://postgresapp.com)/[Others](https://wiki.postgresql.org/wiki/Detailed_installation_guides))
 
-    ### Debian and Fedora dependencies
+    ### Ensure Postgres CLI tools installed
+
+    If you're using [Postgres.app](https://postgresapp.com) on macOS make sure
+    [Postgres CLI tools](https://postgresapp.com/documentation/cli-tools.html) are installed
+
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+      echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
+    ```
+
+    There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
+
+    ## Linux dependencies
```

Comment:
Are these for Crystal? Or things Lucky needs if you're running on Linux?

Reply:
Both I think...I don't know though..

I'm guessing these should be installed before anything else so I'll move it to the top